Kernel/Net/NetworkAdapter.h

```diff
@@ -23,6 +23,8 @@ namespace Kernel {
 
 class NetworkAdapter;
 
+using NetworkByteBuffer = AK::Detail::ByteBuffer<1500>;
```

Comment:
it will break if you use jumbograms

Reply:
Well, if by "break" you mean "slow down significantly" - sure. Other than that it'd still work. I'm not sure that's worth fixing at this point.